lib/ts/recipe/emailpassword/components/themes/resetPasswordUsingToken/resetPasswordEmail.tsx

```diff
@@ -40,18 +42,29 @@ const EmailPasswordResetPasswordEmail: React.FC<EnterEmailProps> = (props) => {
     };
     const { formFields } = props;
 
+    const emailSuccessText =
+        t("EMAIL_PASSWORD_RESET_SEND_BEFORE_EMAIL") +
+        (emailFieldValue !== undefined && emailFieldValue.length > 0
+            ? emailFieldValue
+            : t("EMAIL_PASSWORD_RESET_SEND_FALLBACK_EMAIL")) +
+        t("EMAIL_PASSWORD_RESET_SEND_AFTER_EMAIL");
+
     if (status === "SENT") {
         return (
             <div data-supertokens="container" css={styles.container}>
                 <div data-supertokens="row" css={styles.row}>
                     <div
                         data-supertokens="primaryText enterEmailSuccessMessage"
                         css={[styles.primaryText, styles.enterEmailSuccessMessage]}>
-                        {t("EMAIL_PASSWORD_RESET_SEND_SUCCESS")}
-                        <span data-supertokens="link" css={styles.link} onClick={resend}>
+                        {emailSuccessText}
```

Comment:
Need to test:
- Responsiveness on mobile devices
- What happens if you enter a very long email?